heron/stmgr/src/cpp/manager/stmgr-client.cpp

```diff
@@ -211,13 +219,17 @@ void StMgrClient::StartBackPressureConnectionCb(Connection* _connection) {
   // Ask the StMgrServer to stop consuming. The client does
   // not consume anything
 
-  client_manager_->StartBackPressureOnServer(other_stmgr_id_);
+  if (!droptuples_upon_backpressure_) {
+    client_manager_->StartBackPressureOnServer(other_stmgr_id_);
```

Comment:
I don't know this part of code yet. Just to confirm, the StartBackPressureOnServer() call doesn't affect the HasCausedBackpressure() result, right?

Reply:
Nopes